backend/src/apiserver/server/api_converter.go

```diff
@@ -42,9 +42,9 @@ func ToApiExperimentV1(experiment *model.Experiment) *apiV1beta1.Experiment {
 	}
 	storageState := apiV1beta1.Experiment_StorageState(apiV1beta1.Experiment_StorageState_value["STORAGESTATE_UNSPECIFIED"])
 	switch experiment.StorageState {
-	case "AVAILABLE":
+	case "AVAILABLE", "STORAGESTATE_AVAILABLE":
```

Comment:
Can you please add UTs that cover both case?

Reply:
Sure, I will do that.